prow/spyglass/lenses/buildlog/lens.go

```diff
@@ -116,6 +122,24 @@ type BuildLogsView struct {
 	RawGetMoreRequests map[string]string
 }
 
+func getHighlightRegex(config interface{}) *regexp.Regexp {
+	var c Config
+	if err := mapstructure.Decode(config, &c); err != nil {
```

Comment:
Why not `json.RawMessage` and explicit decoding?

Reply:
Sure, sounds good to me.